website/static/js/comment.js

```diff
@@ -291,6 +511,50 @@ var CommentModel = function(data, $parent, $root) {
         self.author = self.$root.author;
     }
 
+    self.editableContent = ko.computed(function() {
+        var content = self.content();
+        if (!content) {
+            content = '';
+        }
+        var regex = /\[(@|\+)(.*?)\]\(\/([a-z\d]{5})\/\)/;
+        var matches = content.match(/\[(@|\+)(.*?)\]\(\/([a-z\d]{5})\/\)/g);
```

Comment:
As with https://github.com/leb2dg/osf.io/pull/14#discussion_r61789514, can we [DRY](https://en.wikipedia.org/wiki/Don%27t_repeat_yourself) up these two lines?